tests/puppeteer/pages/BO/international/currencies/index.js

```diff
@@ -55,7 +50,7 @@ module.exports = class Currencies extends BOBasePage {
   }
 
   /**
-   * Go to currencies page
+   * Go to localization page
```

Comment:
Are you sure? Since this file is in the `currencies` it would seem logical that it goes to currencies page no? (`this.localizationNavItemLink = '#subtab-AdminCurrencies';`)
Or the maybe the `localizationNavItemLink` is not correct and should be `#subtab-AdminParentLocalization` ?

Reply:
this comment is correct, when its in currencies page, there is 3 others sub-tabs (localization, languages and geolocalization)

Reply:
Yes I noticed, but the localization id is `subtab-AdminLocalization` however here the `localizationNavItemLink` value is `#subtab-AdminCurrencies` So the comment is correct but the selector seems to match the wrong tab no?

Reply:
It was fixed [here](https://github.com/PrestaShop/PrestaShop/pull/16963/commits/c16c7e28d6f9d66ed7ecf0042be20e0691802fdd)

Reply:
It's fixed in `languages/index.js` (in the other PR) But here this is the `currencies/index.js` which has the same mistake (probably copy pasted)

Reply:
My mistake, fixed
thanks